FirebaseInAppMessaging/Sources/DefaultUI/FIRIAMDefaultDisplayImpl.m

```diff
@@ -56,14 +53,23 @@ + (NSBundle *)getViewResourceBundle {
   Class myClass = [self class];
 
   dispatch_once(&onceToken, ^{
+    NSString *bundledResource;
+
+    // When using SPM, Xcode scopes resources to a target, creating a bundle.
+#if SWIFT_PACKAGE
+    bundledResource = @"Firebase_FirebaseInAppMessaging_iOS";
+#else
+    bundledResource = @"InAppMessagingDisplayResources";
+#endif  // SWIFT_PACKAGE
+
     NSBundle *containingBundle;
     NSURL *bundleURL;
     // The containing bundle is different whether FIAM is statically or dynamically linked.
     for (containingBundle in @[ [NSBundle mainBundle], [NSBundle bundleForClass:myClass] ]) {
```

Comment:
What happens for tvOS?  Perhaps add a comment.

Reply:
tvOS doesn't have a default display implementation right now-- so this class isn't included.

Reply:
+1 for comment, though.